### PR TITLE
:arrow_up::one::two: access_apps

### PR DESCRIPTION
**Pull request by RafiZz**

### CODE REVIEW

File: access_apps/README.rst [hunk 1]
```diff
@@ -10,6 +10,8 @@ Adds **Apps access** selection in user's access rights tab. Two options are avai
 * *Allow install apps*
 * *Allow install apps only from settings*
 
+Note: During porting to 12 version, a bug was detected with the selected option "Allow to install apps only from settings" (A user could not open a menu [[ Sales ]] >> Configuration >> Settigns). The error has not been fixed yet.
```
[the remark this answers is not in the document]
Comment (ufaks) [reply access_apps/README.rst]: it's fixed in version 10, but fixes not ported in 12 version

File: access_apps/security/access_apps_security.xml [hunk 2]
```diff
@@ -6,11 +6,11 @@
     <field name="sequence">18</field>
   </record>
   <record id="group_allow_apps_only_from_settings" model="res.groups">
-    <field name="name">Allow install apps only from settings</field>
+    <field name="name">Allow to install apps only from settings</field>
     <field name="category_id" ref="module_category_access_apps"/>
   </record>
   <record id="group_allow_apps" model="res.groups">
-    <field name="name">Allow install apps</field>
+    <field name="name">Allow to install apps</field>
```
Comment (yelizariev) [inline access_apps/security/access_apps_security.xml]: Allow installing apps :policeman: 

grammarly.com says:

The verb <b>install</b> is&nbsp;usually in&nbsp;the gerund form when following the word <b>Allow</b>. Consider replacing it&nbsp;with the <b>-ing</b> form.

**Comment by yelizariev** [comment]
Please remake commits (via ``git rebase -i``) and make two commits only:

* access_apps
* postgresql must be 9.5+

**Comment by yelizariev** [comment]
Very good

**Comment by yelizariev** [comment]
Actually, not ideal.

It's still red, because you haven't configured GPG keys. Please do and make ``git rebase upstream/master`` to re-sign

**Comment by yelizariev** [comment]
Perfect

**Pull request merged**